bin/pubsub_cli.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Pub/Sub manipulation script.
+#
+# Note: Run this in the base container which has access to Pub/Sub.
+#
+# Usage: ./bin/pubsub_cli.py [SUBCOMMAND]
+
+import click
+from google.cloud import pubsub_v1
+from google.api_core.exceptions import AlreadyExists, NotFound
+
+
+@click.group()
+def pubsub_group():
+    """Local dev environment Pub/Sub emulator manipulation script."""
+
+
+@pubsub_group.command("list_topics")
+@click.argument("project_id")
+@click.pass_context
+def list_topics(ctx, project_id):
```

Comment:
How does one know what the project id is?

Reply:
it can be anything, it just needs to match what was used when creating the topics